sirepo/template/activait.py

```diff
@@ -299,6 +299,35 @@ def stateful_compute_compute_column_info(data):
     return _compute_column_info(data.args.dataFile)
 
 
+def stateful_compute_sample_images(data):
+    import matplotlib.pyplot as plt
+    from base64 import b64encode
+    from pykern.pkcompat import from_bytes
+
+    def _data_url(filename):
+        f = open(filename, "rb")
+        u = "data:image/jpeg;base64," + from_bytes(b64encode(f.read()))
+        f.close()
+        return u
+
+    with h5py.File(_filepath("CIFAR-4.h5"), "r") as f:
+        x = f["images"]
```

Comment:
Looks like a hard-coded datafile - this should probably come from a call to lb_file_name_with_model_field() similar to other calls in template/activait.py

Reply:
I'll add that the `DataReaderFactory` is designed to simplify getting the data from archive files so you can avoid opening them directly - I think it can be used here.